mne_bids_pipeline/_config.py

```diff
@@ -1979,24 +1979,33 @@ def mri_landmarks_kind(bids_path):
 
 # ## Inverse solution
 
-loose: float | Literal["auto"] = 0.2
+loose: Annotated[float, Interval(ge=0, le=1)] = 0.2
```

Comment:
To restore old `auto` behavior, something like
```suggestion
loose: Annotated[float, Interval(ge=0, le=1)] | Literal["auto"] = 0.2
```

Reply:
right, forgot about that one!!